src/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X/RelativePathCodeDocumentProcessor.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Razor.Language;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X
+{
+    internal class RelativePathCodeDocumentProcessor : RazorProjectEngineFeatureBase, IRazorCodeDocumentProcessor
```

Comment:
Technically wouldn't need this at all if we had an understanding of "identity" or relative path on the source document

Reply:
Yeah, look at my recent history. I think that the functionality built into this class needs to be the default for all Razor.

Reply:
I don't like this API, and I don't think we need dis.